tomland.cabal

```
@@ -110,3 +110,34 @@ test-suite tomland-test
   default-extensions:  LambdaCase
                        OverloadedStrings
                        RecordWildCards
+
+benchmark tomland-benchmark
+  type:               exitcode-stdio-1.0
+  hs-source-dirs:     benchmark
+  main-is:            Main.hs
+
+  other-modules:      Benchmark.TomlData
+
+  ghc-options:        -Wall
+                      -threaded
+                      -rtsopts
+                      -with-rtsopts=-N
+                      -02
```

This will fix the CI

```suggestion
                      -O2
```

---

Do you mean to delete `-02`